### DISCUSSION TITLE
Dynamically rewrite items href

### DISCUSSION THREAD

**Discussion by edrodrigues-ag** [stac-utils/stac-fastapi-pgstac#129] [Q&A]
Where is the most appropriate place to rewrite href´s of assets? I ingested some items to the database, but their references are local. I want to expose those paths in a http server, but I don´t want to hardwire the paths in the DB. Does that make sense? Thanks anyways.

**Comment by vincentsarago** [comment]
@edrodrigues-ag maybe the easiest would be to create a custom `client` based on `pgstac.core.CoreCrudClient` 

https://github.com/stac-utils/stac-fastapi-pgstac/blob/main/stac_fastapi/pgstac/core.py#L140-L249

where you'll update each items 🤷 

or maybe this is something that can be done with `hydration`
